openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EmailTemplateRepository.java

```diff
@@ -0,0 +1,105 @@
+package org.openmetadata.service.jdbi3;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.openmetadata.schema.auth.EmailTemplate;
+import org.openmetadata.service.util.EmailTemplateTypeDefinition.EmailTemplateType;
+import org.openmetadata.service.util.JsonUtils;
+
+@Slf4j
+public class EmailTemplateRepository {
+  private static CollectionDAO dao;
+
+  public EmailTemplateRepository(CollectionDAO dao) {
+    EmailTemplateRepository.dao = dao;
+  }
+
+  public String readFileContent(String basePath, String fileName) throws IOException {
+    File file = new File(basePath + fileName);
+    StringBuilder fileContent = new StringBuilder();
+    BufferedReader bufferedReader;
+    String readLine;
+    if (file.exists()) {
+      bufferedReader = new BufferedReader(new FileReader(basePath + fileName));
+    } else {
+      file = new File("");
+      String path = file.getAbsolutePath();
+      String[] rootPath = path.split("openmetadata-dist/");
+      bufferedReader =
+          new BufferedReader(
+              new FileReader(rootPath[0] + "/openmetadata-service/src/main/resources/emailTemplates/" + fileName));
+    }
+    while ((readLine = bufferedReader.readLine()) != null) {
+      fileContent.append(readLine);
+    }
+    return fileContent.toString();
+  }
+
+  public void populateTemplateInDb(List<String> fileNames, String basePath) {
+    for (String fileName : fileNames) {
+      try {
+        switch (fileName) {
```

Comment:
why we need to have this switch statement? why not loop through and just call the storeEmailTemplate

Reply:
Addressed in the latest commit.